### PR TITLE
Add `--php-binary` & `--concurrent-processes`

### PR DESCRIPTION
**Pull request by Hywan**
The `--php-binary` value is copied to the `--php` option of atoum.
The `--concurrent-processes`  value is copied to the `--max-children-number` option of atoum.

### CODE REVIEW

File: Bin/Run.php [hunk 1]
```diff
@@ -306,6 +325,8 @@ public function usage()
                 'F'    => 'Filter tests with a ruler expression (see ' .
                           'Hoa\Ruler).',
                 'D'    => 'Activate the debugging mode.',
+                'p'    => 'Path a specific PHP binary.',
```
Comment (jubianchi) [inline Bin/Run.php]: "Path to a specific PHP binary."

**Comment by jubianchi** [comment]
:+1: I just found a tiny typo :)

:shipit:

**Comment by Hywan** [comment]
Damn, I merged it before commiting the fix…

**Comment by Hywan** [comment]
https://github.com/hoaproject/Test/commit/24f93327f29e871a3f824cb38733de95d232fb06

**Comment by Hywan** [comment]
Thanks for the review :-).

**Pull request merged**